### PR TITLE
feat: improve accessibility

### PR DESCRIPTION
**Pull request by YuriDevAT**
## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

## This PR adds/fixes

accessibility failures like missing **alt text** and **low color contrast**.

## Notes

Unfortunately, I was not able to find the scss file to fix color contrast as I wished to do and to fix it everywhere needed. I am sure that changing color with the style attribute was not how this project wanted to handle style changes. I would be happy if I can get information about where to find the stylesheet.

I added an `alt` tag for the GitHub profile `img` to the HTML cell in the yml file. (Deprecated:_Also, since I was not able to "run" the project locally I can only assume my changes on how to add an **alt tag** to the images added via JSON are right._) I am happy if someone with more experience can make sure that the alt tag is added to every image, even if that means, the contribution would not come with my PR now.

### CODE REVIEW

File: generate.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ fs.readdir(
       const profile = require(path.join(__dirname, "profiles", file.name));
 
       profiles.push(profile);
+
+      profiles.image.alt = "Github profile from "`${profiles.name}`;
```
Comment (Panquesito7) [inline generate.js:23]: You might want to capitalize the `h` here. What do you think?
```suggestion
      profiles.image.alt = "GitHub profile from "`${profiles.name}`;
```

**Review by krshdev — APPROVED** [review]

**Review by schmelto — APPROVED** [review]
LGTM

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by vikasganiga05 — COMMENTED** [review]
LGTM :+1:

**Review by Susmita-Dey — APPROVED** [review]
LGTM! 👌

**Pull request merged**